src/store/reducers/cart/types.js

```diff
@@ -0,0 +1,3 @@
+export const ADD_TO_CART = 'ADD_TO_CART'
+export const DELETE_FROM_CART = 'DELETE_FROM_CART'
+export const CHANGE_QUANTITY_IN_THE_CART = 'CHANGE_QUANTITY_IN_THE_CART'
```

Comment:
I was speaking about renaming this, so it has the same name as the action creator function.